tests/manager/api/test_circulationapi.py

```diff
@@ -183,15 +178,6 @@ def test_attempt_borrow_with_existing_remote_loan(
         a loan for.
         """
         # Remote loan.
-        circulation_api.circulation.add_remote_loan(
-            circulation_api.pool.collection,
-            circulation_api.pool.data_source,
-            circulation_api.identifier.type,
-            circulation_api.identifier.identifier,
-            self.YESTERDAY,
-            self.IN_TWO_WEEKS,
-        )
-
```

Comment:
This call to `add_remote_loan` was here, but because we never call `patron_activity`, it makes no difference. So we just remove them without impacting the tests.

A number of these are removed. I'm just going to call out this one, but they were all removed for the same reason.